docs/en/compute-edition/31/rn/book_point_release.yml

```diff
@@ -2,7 +2,7 @@
 kind: book
 title: Prisma Cloud Compute Edition Release Notes
 author: Prisma Cloud team
-version: 31.02
+version: 31.03
```

Comment:
@rcaballeromx These changes are already merged in main, please refer to the PR over here